src/clients/add-on_auto.rb

```diff
@@ -194,8 +194,11 @@ def main
                   )
                 )
               else
-                # just report error
-                Report.Error(_("Failed to add add-on product."))
+                # just report an error
+                error_string = format(_("Failed to add product \"%s\" via\n%s\n"),
+                  prod["product"], media)
+                error_string << _("Please check logfiles for more information.")
```

Comment:
I'm not sure it makes sense, I'd rather remove this part. Either the user knows where YaST logs all details and then it's useless or the user does not know and then it does not help at all.

Experts already know that, inexperienced users should not try reading the log at all. :wink: